packages/core/src/util/requireHook.js

```diff
@@ -54,6 +54,25 @@ exports.init = function (config) {
  * @param {string} moduleName
  */
 function patchedModuleLoad(moduleName) {
+  let module = moduleName;
```

Comment:
_very nitpicky nitpick:_ I feel `moduleName` is still the better variable name. When I read `module` in the context of what we are doing in the require hook, I think it would refer to the actual module object that is returned by `require`. Is there a particular reason for introducing a new variable? We could reassign the `moduleName` in the code below, could we?

Reply:
👍 Will use the original var